.gitignore

```diff
@@ -34,3 +34,6 @@ cassettes/
 
 # Coverage
 *.lcov
+
+# Imported linting config
+easypost_java_style.xml
```

Comment:
suggestion: I don't feel this is necessary to add, just reference the new one from examples